recipe/meta.yaml

```diff
@@ -81,7 +81,7 @@ outputs:
         - psutil
         - pyyaml
         - requests
-        - setproctitle
+        - setproctitle==1.2.2
```

Comment:
I don't see this in the patch for setup.py - where is this pin coming from?

Reply:
[here](https://github.com/ray-project/ray/blob/840215bc09e942b50cad0ab2db96a8fdc79217c1/python/setup.py#L538)

```
 pip_packages = ["psutil", "setproctitle==1.2.2", "colorama"]
```